migrations/000001_todo_api.down.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
+
```

Comment:
burada begin ve commit kullanmış begin ve commit ne için kullanılıyor kullnmazsak artısı veya eksisi nedir?